content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_control-plane.md

```diff
@@ -0,0 +1,100 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference conent, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+
+Atualiza a instância do plano de controle implantada nesse nó, se houver
+
+### Sinopse
+
+Atualiza a instância do plano de controle implantada nesse nó, se houver
```

Comment:
Suggestion: "instalada" instead of "implantada".

Let's use "camada de gerenciamento" for control plane here too.